src/AccessibilityInsights.SharedUxTests/Settings/ConfigurationModelTests.cs

```diff
@@ -141,6 +142,7 @@ public void LoadFronJSON_CurrentFormat_DataIsCorrect()
             ConfigurationModel config = ConfigurationModel.LoadFromJSON(@".\Resources\ConfigSettings.json", testProvider);
 
             ConfirmOverrideConfigMatchesExpectation(config,
+                colorContrastVersionThresholdSize: 20000,
```

Comment:
```suggestion
                colorContrastVersionThresholdSize: 10000,
```

Reply:
This is intentionally not 10000. This test reads a file that includes _non-default_ values as a proof that the file loading code is properly handling the JSON.